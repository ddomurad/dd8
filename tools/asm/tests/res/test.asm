; this is an nonsense asm program
; used only for testing the final binary output

php
php ; some test comment

ldai 0x10         ; load 16
ldai 16           ; load 16
ldai 1_00       ; load 16
ldai 0b00010000   ; load 16
ldai 0b0001_0000  ; load 16

lda 0x55   ; zerop page load 
lda 0x1111 ; absolute loade 
lda 0x41, x 
lda 0x4100, x 
lda 0x1133, y

