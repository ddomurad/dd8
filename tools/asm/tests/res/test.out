:160000000808A910A910A964A910A910A555AD1111B541BD00412C
:00000001FF
